src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -12,105 +12,325 @@ Sentry's Performance features are currently in beta. For more details about acce
+
+When your backend processes the requests from A's browser, it sees the "yes" decision, collects transaction and span data, and sends it to Sentry. Further, it includes the "yes" decision in any requests it makes to subsequent services (like your database server), which similarly collect data, send it to Sentry, and pass the decision along to any services they call. Through this process, all of the relevant transactions in A's trace are collected and sent to Sentry.
+
+On the other hand, when your backend processes the requests from B's browser, it sees the "no" decision, and as a result it does _not_ collect and send transaction and span data to Sentry. It does, however, do the same thing it does in A's case in terms of propagating the decision to subsequent services, telling them not to collect or send data either. They then in turn tell any services they call not to send data, and in this way no transactions from B's trace are collected.
+
+Put simply: as a result of this head-based approach, where the decision is made once in the originating service and passed to all subsequent services, either all of the transactions for a given trace are collected, or none are, so there shouldn't be any incomplete traces.
+
+### Consistency Between Traces
+
+If you enable tracing in services with multiple entry points, we recommend choosing similar sampling rates, to avoid biasing your data. For example, suppose the backend of our on-going webapp example also serves as a public API. In that case, some traces would start with a pageload transaction in the web app, and likely include multiple backend transactions, while other traces (those representing folks hitting the public API) would begin with a single backend request transaction, which would be the only backend transaction in the trace. Choosing a very different sampling rate for your web app from that chosen for your backend would lead to one of those scenarios being oversampled compared to the other, compromising the accuracy of your overall data.
+
+## Viewing Trace Data
+
+You can see a list of transaction events by clicking on the "Transactions" pre-built query in [Discover]({%- link _documentation/performance/discover/index.md -%}), or by using a search condition `event.type:transaction` in the [Discover Query Builder]({%- link _documentation/performance/discover/query-builder.md -%}) view.
+
+### Transaction List View
+
+The results of either of the above queries are presented in a list view, where each entry represents a group of one or more transactions. Data about each group is displayed in table form, and comes in two flavors: value-based (such as transaction name), and aggregate (such as average duration). The choice of which kinds of data to display is configurable, and can be changed by clicking 'Edit Columns' at the top right of the table. Bear in mind that adding or removing any value-based columns may affect the way the results are grouped.
```

Comment:
This is actually one of 3 major views that the above query will show. 
1) The list view.
2) A timeseries graph showing the aggregates of the query over time.
3) A tag summary that shows the top tags associated with the events in the list view.

I think it's worth at least mentioning the timeseries graph, since there are a bunch of interesting features relevant to tracing: you can see any of the aggregates in your query graphed over time, overlay the results from the previous period or release markers, and see a breakdown of the top 5 results in the list view.

Reply:
Okay. I was hoping to avoid going into too much detail here (since the full Discover interface is covered in the linked docs), but I'm happy to add a sentence or two.

Reply:
Oh I see your point. OK well if you want to keep it terse here I think that's fine. I do think the timeseries graph is very useful but I do know it's documented separately.

Reply:
I added a few sentences, and mentioned both the graph and the tag summary.